tests/award.js

```diff
@@ -377,6 +377,6 @@ module.exports = {
                         done();
                     })
             })
-            .end();
+            //.end();
     }
```

Comment:
Same comment here, does the test finish without the end() or does the session hang?